src/set_theory/game/pgame.lean

```diff
@@ -586,26 +586,31 @@ theorem equiv_congr_right {x₁ x₂} : x₁ ≈ x₂ ↔ ∀ y₁, x₁ ≈ y
  λ h, (h x₂).2 $ equiv_rfl⟩
 
 theorem equiv_of_mk_equiv {x y : pgame}
-  (L : x.left_moves ≃ y.left_moves) (R : x.right_moves ≃ y.right_moves)
-  (hl : ∀ (i : x.left_moves), x.move_left i ≈ y.move_left (L i))
-  (hr : ∀ (j : y.right_moves), x.move_right (R.symm j) ≈ y.move_right j) :
-  x ≈ y :=
+  (L : x.left_moves ≃ y.left_moves) (R : y.right_moves ≃ x.right_moves)
+  (hl : ∀ i, x.move_left i ≈ y.move_left (L i))
+  (hr : ∀ j, x.move_right (R j) ≈ y.move_right j) : x ≈ y :=
 begin
   fsplit; rw le_def,
-  { exact ⟨λ i, or.inl ⟨L i, (hl i).1⟩, λ j, or.inr ⟨R.symm j, (hr j).1⟩⟩ },
+  { exact ⟨λ i, or.inl ⟨L i, (hl i).1⟩, λ j, or.inr ⟨R j, (hr j).1⟩⟩ },
   { fsplit,
-    { intro i,
-      left,
+    { refine λ i, or.inl _,
       specialize hl (L.symm i),
       simp only [move_left_mk, equiv.apply_symm_apply] at hl,
       use ⟨L.symm i, hl.2⟩ },
-    { intro j,
-      right,
-      specialize hr (R j),
-      simp only [move_right_mk, equiv.symm_apply_apply] at hr,
-      use ⟨R j, hr.2⟩ } }
+    { refine λ j, or.inr _,
+      specialize hr (R.symm j),
+      simp only [move_right_mk, equiv.apply_symm_apply] at hr,
+      use ⟨R.symm j, hr.2⟩ } }
 end
 
+/-- The same as `pgame.equiv_of_mk_equiv` but with the equivalences swapped. -/
+theorem equiv_of_mk_equiv' {x y : pgame}
```

Comment:
I don't see the necessity of this lemma. It can be proved from `equiv_of_mk_equiv` using symmetry of ≈ in `hl`, `hr` and `⊢`. And you didn't use the primed versions (including `quot_eq_of_mk_quot_eq'`) to prove anything new, so there doesn't seem to be strong reason in introducing them. I see you used `mk'` to remove some `simpa` though.

Reply:
I added this lemma as a sort of afterthought to match `mk'`, will gladly remove.